src/black/const.py

```diff
@@ -1,4 +1,19 @@
+from enum import Enum
+from typing import Any
+
 DEFAULT_LINE_LENGTH = 88
 DEFAULT_EXCLUDES = r"/(\.direnv|\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|venv|\.svn|_build|buck-out|build|dist)/"  # noqa: B950
 DEFAULT_INCLUDES = r"(\.pyi?|\.ipynb)$"
 STDIN_PLACEHOLDER = "__BLACK_STDIN_FILENAME__"
+
+
+class OutputLevels(Enum):
+    null: Any = dict()
```

Comment:
As discussed, I prefer literals. This way the flake8 config doesn't have to change as well. I'll include it here as well so that others may comment.

Reply:
Also `none` could be a more pythonic name.

Reply:
`None` would be more pythonic agreed, but I would need to add some additional logic to `_out` function, I wanted to keep it simple hence using am using an empty `dict` here.

Regarding the `dict` literals, I am using function calls here as I found them more readable compared to literals.

Reply:
I meant `none: dict = {}`!

Reply:
We should either use dict literals here or decide that we globally don't care for this flake8 setting and disable it.

Reply:
> I meant `none: dict = {}`!

Ah, that would work 👍🏻 (the name, mypy complains about `: dict` as it needs the parameters)

Reply:
The Flake8 rule is definitely useful. So maybe we'll sacrifice a bit of readability and use literals here.

Reply:
I am not sure, either way is fine. We could even safely ignore the rule since very few people use functional calls over literals for normal use cases (which can be caught in reviews), otherwise, they are only used for some specific reasons like readability/explicitness (this case).